Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -492,7 +492,10 @@ The following sizes for ``M`` are supported as ``maglev.tableSize`` Helm option:
 
 For example, a ``maglev.tableSize`` of ``16381`` is suitable for a maximum of ``~160`` backends
 per service. If a higher number of backends are provisioned under this setting, then the
-difference in reassignments on backend changes will increase.
+difference in reassignments on backend changes will increase. Note that changing the table
+size (``M``) triggers a recalculation of the lookup table and can temporarily lead to inconsistent
+backend selection for new traffic until all nodes have converged and completed their agent
+restart respectively bootstrap.
```

Comment:
I'm struggling with the end of this sentence, what's `their agent restart respectively bootstrap`? Did you mean restart and bootstrap?

Reply:
yes, will fix